### ISSUE TITLE
unexpected EOF (Known) Issue 

### ISSUE THREAD

**Issue by gsfernandes81** [IceDynamix/reliquary-archiver#1]
Hi! I encountered the unexpected EOF issue that you mentioned in the release notes and was looking to provide logs if helpful:

The following are the verbose logs from the archiver:

```
PS C:\Users\<redacted>\Downloads> .\reliquary-archiver_v0.1.3.exe -v
2024-04-08T11:40:55.097360Z  INFO config_map: reliquary_archiver::export::fribbels: initializing database from online sources, this might take a while...
2024-04-08T11:41:00.749961Z  INFO live_capture: reliquary_archiver: instructions: go to main menu screen and go into train hyperdrive
2024-04-08T11:41:00.750122Z  INFO live_capture: reliquary_archiver: listening with a timeout of 120 seconds...
2024-04-08T11:41:21.895623Z  WARN live_capture:receive_packet{len=62}:parse_connection_packet: reliquary::network::connection: disconnected packet
2024-04-08T11:41:21.896481Z  WARN live_capture:receive_packet{len=62}:parse_connection_packet: reliquary::network::connection: disconnected packet
2024-04-08T11:41:47.903126Z  INFO live_capture:receive_packet{len=62}:parse_connection_packet: reliquary::network::connection: handshake requested
2024-04-08T11:41:47.903428Z  INFO live_capture:receive_packet{len=62}: reliquary::network: handshake requested, resetting state
2024-04-08T11:41:47.903620Z  INFO live_capture:receive_packet{len=62}:parse_connection_packet: reliquary::network::connection: handshake requested
2024-04-08T11:41:47.903760Z  INFO live_capture:receive_packet{len=62}: reliquary::network: handshake requested, resetting state
2024-04-08T11:41:47.931220Z  INFO live_capture: reliquary_archiver: detected connection established
2024-04-08T11:41:47.931393Z  INFO live_capture: reliquary_archiver: detected connection established
2024-04-08T11:41:47.953329Z  INFO live_capture:receive_packet{len=145}:try_new:new{conv_id=824870}: reliquary::network::kcp: new connection, created new kcp instance
2024-04-08T11:41:47.953555Z  INFO live_capture:receive_packet{len=145}:receive_command{len=75}: reliquary::network::crypto: found initial decryption key version=315284530
2024-04-08T11:41:47.953725Z  INFO live_capture:receive_packet{len=145}:receive_command{len=75}:command{command=GameCommand { command_id: 16, command_name: Some("PlayerGetTokenCsReq"), header_len: 2, data_len: 57 }}: reliquary::network: received
2024-04-08T11:41:48.008084Z  INFO live_capture:receive_packet{len=70}:try_new:new{conv_id=824870}: reliquary::network::kcp: new connection, created new kcp instance
2024-04-08T11:41:48.213204Z  INFO live_capture:receive_packet{len=105}:receive_command{len=35}:command{command=GameCommand { command_id: 60, command_name: Some("PlayerGetTokenScRsp"), header_len: 0, data_len: 19 }}: reliquary::network: received
2024-04-08T11:41:48.213754Z  INFO live_capture:receive_packet{len=105}:receive_command{len=35}:command{command=GameCommand { command_id: 60, command_name: Some("PlayerGetTokenScRsp"), header_len: 0, data_len: 19 }}: reliquary::network: setting new session key seed=17741017250029361150
2024-04-08T11:41:48.241864Z  INFO live_capture:receive_packet{len=986}:receive_command{len=916}:command{command=GameCommand { command_id: 10, command_name: Some("PlayerLoginCsReq"), header_len: 2, data_len: 898 }}: reliquary::network: received
2024-04-08T11:41:48.352219Z  INFO live_capture:receive_packet{len=142}:receive_command{len=72}:command{command=GameCommand { command_id: 33, command_name: Some("PlayerLoginScRsp"), header_len: 0, data_len: 56 }}: reliquary::network: received
2024-04-08T11:41:48.352879Z  INFO live_capture: reliquary_archiver: detected login
2024-04-08T11:41:48.354550Z  INFO live_capture:receive_packet{len=1178}:receive_command{len=12385}:command{command=GameCommand { command_id: 23, command_name: None, header_len: 12, data_len: 12357 }}: reliquary::network: received
2024-04-08T11:41:48.354986Z  INFO live_capture:receive_packet{len=1178}:receive_command{len=28}:command{command=GameCommand { command_id: 31, command_name: Some("UpdateFeatureSwitchScNotify"), header_len: 12, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.355348Z  INFO live_capture:receive_packet{len=1178}:receive_command{len=28}:command{command=GameCommand { command_id: 1478, command_name: Some("SyncServerSceneChangeNotify"), header_len: 12, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.355653Z  INFO live_capture:receive_packet{len=1178}:receive_command{len=28}:command{command=GameCommand { command_id: 586, command_name: Some("SyncTurnFoodNotify"), header_len: 12, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.356054Z  INFO live_capture:receive_packet{len=1178}:receive_command{len=38}:command{command=GameCommand { command_id: 81, command_name: Some("StaminaInfoScNotify"), header_len: 12, data_len: 10 }}: reliquary::network: received
2024-04-08T11:41:48.356544Z  INFO live_capture:receive_packet{len=1178}:receive_command{len=35}:command{command=GameCommand { command_id: 1226, command_name: Some("DailyTaskDataScNotify"), header_len: 12, data_len: 7 }}: reliquary::network: received
2024-04-08T11:41:48.357002Z  INFO live_capture:receive_packet{len=1178}:receive_command{len=28}:command{command=GameCommand { command_id: 2216, command_name: Some("RaidInfoNotify"), header_len: 12, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.357441Z  INFO live_capture:receive_packet{len=1178}:receive_command{len=62}:command{command=GameCommand { command_id: 3010, command_name: Some("BattlePassInfoNotify"), header_len: 12, data_len: 34 }}: reliquary::network: received
2024-04-08T11:41:48.412449Z  INFO live_capture:receive_packet{len=88}:receive_command{len=18}:command{command=GameCommand { command_id: 94, command_name: Some("GetLevelRewardTakenListCsReq"), header_len: 2, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.418082Z  INFO live_capture:receive_packet{len=289}:receive_command{len=35}:command{command=GameCommand { command_id: 583, command_name: Some("ComposeLimitNumCompleteNotify"), header_len: 12, data_len: 7 }}: reliquary::network: received
2024-04-08T11:41:48.418688Z  INFO live_capture:receive_packet{len=289}:receive_command{len=39}:command{command=GameCommand { command_id: 547, command_name: Some("GeneralVirtualItemDataNotify"), header_len: 12, data_len: 11 }}: reliquary::network: received
2024-04-08T11:41:48.419169Z  INFO live_capture:receive_packet{len=289}:receive_command{len=31}:command{command=GameCommand { command_id: 6242, command_name: Some("StoryLineInfoScNotify"), header_len: 12, data_len: 3 }}: reliquary::network: received
2024-04-08T11:41:48.419612Z  INFO live_capture:receive_packet{len=289}:receive_command{len=30}:command{command=GameCommand { command_id: 2987, command_name: Some("NewAssistHistoryNotify"), header_len: 12, data_len: 2 }}: reliquary::network: received
2024-04-08T11:41:48.470902Z  INFO live_capture:receive_packet{len=169}:receive_command{len=99}:command{command=GameCommand { command_id: 96, command_name: Some("GetLevelRewardTakenListScRsp"), header_len: 12, data_len: 71 }}: reliquary::network: received
2024-04-08T11:41:48.646940Z  INFO live_capture:receive_packet{len=88}:receive_command{len=18}:command{command=GameCommand { command_id: 29, command_name: Some("GetBasicInfoCsReq"), header_len: 2, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.657431Z  INFO live_capture:receive_packet{len=88}:receive_command{len=18}:command{command=GameCommand { command_id: 32, command_name: Some("GetHeroBasicTypeInfoCsReq"), header_len: 2, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.669711Z  INFO live_capture:receive_packet{len=88}:receive_command{len=18}:command{command=GameCommand { command_id: 510, command_name: Some("GetBagCsReq"), header_len: 2, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.680331Z  INFO live_capture:receive_packet{len=88}:receive_command{len=18}:command{command=GameCommand { command_id: 532, command_name: Some("GetMarkItemListCsReq"), header_len: 2, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.691522Z  INFO live_capture:receive_packet{len=88}:receive_command{len=18}:command{command=GameCommand { command_id: 2810, command_name: Some("GetPlayerBoardDataCsReq"), header_len: 2, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.702235Z  INFO live_capture:receive_packet{len=90}:receive_command{len=20}:command{command=GameCommand { command_id: 310, command_name: Some("GetAvatarDataCsReq"), header_len: 2, data_len: 2 }}: reliquary::network: received
2024-04-08T11:41:48.712872Z  INFO live_capture:receive_packet{len=88}:receive_command{len=18}:command{command=GameCommand { command_id: 2932, command_name: Some("GetCurAssistCsReq"), header_len: 2, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.724219Z  INFO live_capture:receive_packet{len=88}:receive_command{len=18}:command{command=GameCommand { command_id: 766, command_name: Some("GetAllLineupDataCsReq"), header_len: 2, data_len: 0 }}: reliquary::network: received
2024-04-08T11:41:48.725719Z  INFO live_capture:receive_packet{len=351}:receive_command{len=49}:command{command=GameCommand { command_id: 24, command_name: Some("GetBasicInfoScRsp"), header_len: 12, data_len: 21 }}: reliquary::network: received
2024-04-08T11:41:48.725926Z  INFO live_capture:receive_packet{len=351}:receive_command{len=148}:command{command=GameCommand { command_id: 72, command_name: Some("GetHeroBasicTypeInfoScRsp"), header_len: 12, data_len: 120 }}: reliquary::network: received
2024-04-08T11:41:48.726085Z  WARN live_capture: reliquary_archiver::export::fribbels: could not parse trailblazer data command error=Unexpected EOF
```

Happy to run any special debug builds of the program on my end if it would be helpful for you. Thanks for your work on this!

**Comment by KamilNiemcewicz** [comment]
I had the same issue - it fixed itself when I run it through command line with those parameters:
![image](https://github.com/IceDynamix/reliquary-archiver/assets/31008031/647a258f-ccab-4d4a-a117-2d905f0e22ac)

Maybe double clicking the executable sets something differently than running it through CMD? 

Worth nothing - the CMD was not run as admin. Maybe then the issue only occur when you run it as admin? I don't know, might be worth testing @gsfernandes81

**Comment by gwendolynemaxinegwenstacy** [comment]
I have the same problem, but i can't find where the logs is. After running it again, it worked.

**Comment by gsfernandes81** [comment]
Thanks for the suggestions folks. I did try it several times at the time, both running from the command line, and double clicking it.
Have run it as the user (have not tried running it as Admin admittedly)

Tried again today though and it got one unexpected EOF but otherwise worked fine and gave me my relics. 

Will leave this issue open in case we find a pattern to the issue or in case someone else sees the same issue and wants to report what they see
